### DISCUSSION TITLE
Little clarifications needed in the documentation

### DISCUSSION THREAD

**Discussion by ongolaboy** [peering-manager/peering-manager#452] [Q&A] [answered]
Hello. I've installed for the first time peering manager today. Thank you for this tool. My platform: debian 10.10 .

Just a few comments regarding [the documentation](https://peering-manager.readthedocs.io/en/stable/)

### Setup the Peering Manager codebase

- In addition to packages to install in the system, I also needed: `rustc`
- There are no packages called `python3-setuputils`.

In the _Create the Peering Manager User_ , [section](https://peering-manager.readthedocs.io/en/stable/setup/3-peering-manager/#create-the-peering-manager-user)
 
Instead of `adduser --system --gid peering-manager peering-manager` it should be `adduser --system --gid GROUPID peering-manager` where GROUPID is the numerical id of the group created at the previous step.

In the _Set Up Python Environment_ [section](https://peering-manager.readthedocs.io/en/stable/setup/3-peering-manager/#set-up-python-environment)

I have to update pip. The one coming with debian buster is pip-18.1 .. It was not good enough to compile everything. So I installed and upgraded one in the virtual environment `pip3 install --upgrade pip`

### Web Server Setup 

Personal choice here ..

- /etc/apache2/sites-enabled/peering-manager.conf`

```
ProxyPass / http://[::1]:8001/
ProxyPassReverse / http://[::1]:8001/`
```

Specific logging file

```
  ErrorLog ${APACHE_LOG_DIR}/peering-manager.error.log
  CustomLog ${APACHE_LOG_DIR}/peering-manager.access.log combined
```
- /opt/peering-manager/gunicorn.py
```
bind = '[::1]:8001'
workers = 5
threads = 3
timeout = 300
max_requests = 5000
max_requests_jitter = 500
user = 'peering-manager'
```

**Comment by jamesditrapani** [comment]
Thanks for your feedback. We currently have a PR open to address some documentation rewrites (#429). We will ensure your points are addressed here!

**Comment by Paktosan** [comment, marked as answer]
Hi, being the person doing #429, I have some comments.
* I can't reproduce the requirement for `rustc`, could you elaborate on that?
* The documentation for required packages will not contain `python3-setuputils` in the future, this is already addressed.
* The `adduser` line will be replaced with `useradd` in the next documentation version.
* Upgrading pip will be in the next documentation version.
* For now I would leave all examples with IPv4 since I have seen systems having IPv6 disabled completely (a shame, but what can I do...).
